amida-tech/amida-notification-microservice
# Merge release/1.2.0 into develop

## To Test:

This release branch (specifically release `1.2.0-rc.3`) has been thoroughly tested by the test team.

The only change in `1.2.0-rc.4` (and therefore the only change that hasn't been tested thoroughly) is the changing of the version in `package.json` from 0.3.0 to 1.2.0.

Due to this change, the API now has `/v1` in it, as in `/api/v1/some-endpoint`.

I deployed this to the testing servers and it worked.

To test, run this service and do a Postman request against an endpoint. It should work normally.
